### ISSUE TITLE
Where can I refer to the code for amodal completion and depth verification for custom datasets?

### ISSUE THREAD

**Issue by hoya-cho** [Championchess/A-Tri-Layer-Plugin-to-Improve-Occluded-Detection#9]
I really enjoyed reading the paper. So, I'm trying to give it a try on custom data, but for amodal completetion, I read a comment saying to use the code in the de-occlusion repo after seeing another issue's post. So, where can I refer to the code for depth verification?

Or is there any code for amodal completion or depth verfication in this repo? If not, did you encounter any difficulties, such as data structure issues, while proceeding with the separate steps?

Thank you for your kind explanation.

**Comment by Championchess** [comment]
Thank you for your interest in our work. You can download the pre-trained depth prediction model from the MiDaS official website and estimate the depth of your custom dataset images. After getting the estimated depth, you could compute the average depth for each instance and then verify their occlusion relations.

**Comment by Championchess** [comment]
There is no data structure issue for me because it is straight forward to get the image depth map prediction and then calculate the average depth for each instance in the image given their masks.

**Comment by hoya-cho** [comment]
thanks!!
